src/File.php

```diff
@@ -144,6 +144,9 @@ class File extends DataObject implements AssetContainer, Thumbnail, CMSPreviewab
         "File" => "DBFile",
         // Only applies to files, doesn't inherit for folder
         'ShowInSearch' => 'Boolean(1)',
+        // FQCN needs 8 backslashes for the database gets a single escaped backslash (i.e. 2 backslashes total).
+        // PHP string dissolves it to 4, Injector::parse_class_spec() dissolves it to 2 which is used in the SQL.
+        'IsFolder' => 'GeneratedColumn("Boolean", "CASE WHEN \\"ClassName\\"=\'SilverStripe\\\\\\\\Assets\\\\\\\\Folder\' THEN 1 ELSE 0 END", "STORED")'
```

Comment:
Change to array syntax

Probably makes sense to keep as STORED since it's something that's also used on the frontend

Reply:
I'm not sure what you mean about being used on the front-end but keeping as stored makes sense to me :+1: